eo-runtime/src/main/eo/org/eolang/positive-infinity.eo

```diff
@@ -181,17 +181,94 @@
   #  worked with multiple arguments without using objects from eo-collections.
   #  See positive-infinity.plus and negative-infinity.plus as an example
   [x...] > div
-    x.at 0 > first!
+    negative-infinity > neg-inf!
+    ^ > pos-inf!
+    0 > zero!
 
     # here we use the rule "NaN != NaN is TRUE" to check if "num" is NaN
-    [num] > check-nan
+    [num] > is-nan
       not. > @
         num.eq num
 
+    [term] > is-nan-or-infinite
+      or. > @
+        is-nan
+          term > t!
+        t.eq pos-inf
+        t.eq neg-inf
+
+    [term] > is-term-gte-zero
+      try > @
+        []
+          term.gte 0 > @
+        [e]
+          term.gte 0.0 > @
+        FALSE
+
+    # @todo #1824:30min. Pass "is-nan-or-infinite" to "div-rec" here and in
+    #  negative-infinity. Now for some reason it's not possible to pass two or
+    #  more objects-functions to another object. To avoid code duplication
+    #  "is-nan-or-infinite" object should be passed to "times-rec" as well as
+    #  "is-term-gte-infinite"
```

Comment:
@maxonfjvipon As far as I understand this todo issue, you want to provide to the object more than 3 arguments. If it's it right you can just add all your arguments to new array and pass this array as argument to particular object

Reply:
@Graur yes, I tried this way but check this out: https://github.com/objectionary/eo/issues/1841

Reply:
@maxonfjvipon I see, let's keep todo